Magic™Editor.user.js

```diff
@@ -3494,21 +3495,20 @@
         // This is the styling for the diff output.
         $('body').append('<style>' +
                          '.difftitle {' +
-                         '    color: rgb(34, 34, 34);' +
+                         '    color: var(--black);' +
                          '    font-size: 24px;' +
                          '    font-weight: normal;' +
                          '    line-height: 36px;' +
                          '    margin-bottom: 12px;' +
                          '}' +
                          '.diffbody {' +
                          '    white-space: pre-wrap;' +
-                         '    font-family: "courier new", "lucida sans typewriter", mono, monospace' +
```

Comment:
Why was this removed rather than replaced? I would expect this to be `Consolas, Menlo, Monaco, Lucida Console, Liberation Mono, DejaVu Sans Mono, Bitstream Vera Sans Mono, Courier New, monospace, sans-serif` to match the site's diff font-family.

Reply:
It's removed to inherit the font-family from the site's CSS, rather than override it. Will change it to `var(--ff-mono)` to match the diff style.